Apps/EGIAmpServer/CMakeLists.txt

```diff
@@ -19,11 +19,13 @@ if(NOT TARGET LSL::lsl)
 endif()
 
 # GENERAL CONFIG #
-set(META_PROJECT_DESCRIPTION "Stream from EGI devices via Lab Streaming Layer")
+set(META_PROJECT_DESCRIPTION "Stream from Philips Neuro devices via Lab Streaming Layer")
 
 # THIRD PARTY LIBRARIES #
 find_package(Qt5 COMPONENTS Widgets REQUIRED)
 find_package(Boost COMPONENTS system REQUIRED)
+find_package(Boost COMPONENTS date_time REQUIRED)
```

Comment:
I don't see any Boost.DateTime / Boost.regex-usage here so we shouldn't need this

Reply:
In Visual Studio, at least, when using Boost.Asio Boost.DateTime and Boost.regex are automatically linked and results in compile time errors if these are not available. I have removed these, though, and instead added a couple compile-time defines (BOOST_DATE_TIME_NO_LIB and BOOST_REGEX_NO_LIB) that eliminate this dependency.